src/Fixer/Comment/HeaderCommentFixer.php

```diff
@@ -198,11 +198,6 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
                 }
```

Comment:
👍🏻 in general,

yet any automation? let's put behind CI

Reply:
I see here
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/8599/files#diff-667a5636268e23a65d461fe5ff15f044b7ebd283c3a626bea2d0aa19fed97b2aR22